vnext/Mso.UnitTests/Mso.UnitTests.vcxproj

```diff
@@ -73,6 +71,7 @@
       <AdditionalOptions>/await %(AdditionalOptions) /bigobj</AdditionalOptions>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard>stdcpp17</LanguageStandard>
+      <CallingConvention>Cdecl</CallingConvention>
```

Comment:
>Cdecl [](start = 25, length = 5)

Should we also set it in Microsoft.ReactNative?

Reply:
I'm not sure it's necessary - the unit tests depend on the google test binaries (via nuget) which requires Cdecl.

Reply:
This is fine, as long as tests pass.

We will move to Cdecl on x86 within the next week (#4096).